src/utils/persistMetrics.js

```diff
@@ -0,0 +1,5 @@
+const { green } = require('../constants/colors');
```

Comment:
I'd rename the file like an interactor named metrics.js

Reply:
I was more like a util function to just persist the metrics in the bd. As the name of the folder is utils. And its function will take care of the integration with the db